info/v2/container.go

```diff
@@ -86,8 +86,11 @@ type ContainerStats struct {
 	// The time of this stat point.
 	Timestamp time.Time `json:"timestamp"`
 	// CPU statistics
-	HasCpu bool        `json:"has_cpu"`
-	Cpu    v1.CpuStats `json:"cpu,omitempty"`
+	HasCpu bool `json:"has_cpu"`
+	// In nanoseconds (aggregated)
+	Cpu v1.CpuStats `json:"cpu,omitempty"`
+	// In nanocores per second (instantaneous)
+	CpuInst v1.CpuStats `json:"cpu_inst,omitempty"`
```

Comment:
I would have assumed that v1 was frozen by now. Otherwise, shouldn't changes in v1 be in v2 too?

Would moving kubelet to v2 be an issue?